htdocs/installdb.php

```diff
@@ -74,7 +74,7 @@
     exit(3);
 }
 
-require_once __DIR__ . "/../vendor/autoload.php";
+require_once __DIR__ . '/../vendor/autoload.php';
```

Comment:
It is for uniformity in the code. Considering the string a literal, there is no justification for using double quotes.

Reply:
According to `git grep autoload.php`, the code nearly uniformly uses double quotes, not single quotes for autoload.php. The only exceptions are the api code and the dataquery module code (and I'm not even sure why the dataquery ajax scripts are requiring it at all, since it should have already been by AjaxHelper.php before it gets to that point..)

Reply:
```
     38 ';
     47 ";
      1 ";
      1 ");
```

Reply:
Nonetheless, I was referring to that file quotation uniformity